app/assets/stylesheets/modules/mixins.scss

```diff
@@ -27,3 +27,14 @@
   padding-left: 3em;
   padding-right: 3em;
 }
+
+@mixin link-border {
+  border-bottom: 2px solid transparent;
+  color: #ffffff;
+  font-family: $dogstay-font-theme;
+}
+
+@mixin hover{
+  border-color: #008080;
```

Comment:
Color literals like `#008080` should only be used in variable declarations; they should be referred to via variable everywhere else.